core/src/main/java/dk/alexandra/fresco/framework/network/Network.java

```diff
@@ -63,4 +72,51 @@ default void sendToAll(byte[] data) {
     }
   }
 
+  /**
+   * Creates a map of party IDs to network address (uses localhost and supplied ports) information.
+   *
+   * @param n the number of parties
+   * @param ports the ports to be used
+   * @return a map of party IDs to network address information
+   */
+  static Map<Integer, NetworkConfiguration> getNetworkConfigurations(int n,
+      List<Integer> ports) {
+    Map<Integer, NetworkConfiguration> confs = new HashMap<>(n);
```

Comment:
The parameter `n` is redundant. Simply use `ports.size()`.

Reply:
Changed